### ISSUE TITLE
Grunt `build` task to use a temp dir before replacing dest dir to reduce server downtime.

### ISSUE THREAD

**Issue by pilwon** [pilwon/ultimate-seed#13]

**Comment by yaru22** [comment]
Isn't it resolved in the current Gruntfile.js?

**Comment by pilwon** [comment]
@yaru22 Not exactly, since `grunt build` completely removes currently serving dist directory before it actually starts the build process. This is suboptimal in the production environment where you don't want any downtime. Ideally it should cleanly build elsewhere and then swap out the existing project only when it's done.
